fawltydeps/settings.py

```diff
@@ -100,21 +100,42 @@ def parse_path_or_stdin(arg: str) -> PathOrSpecial:
 
 DEFAULT_IGNORE_UNUSED = {
     # Development tools not meant to be imported
+    # Formatting Tools
+    "autopep8",
     "black",
+    "isort",
+    "pyformat",
+    "yapf",
+    # Linting Tools
+    "bandit",
```

Comment:
`bandit` is a security tool

Reply:
Thanks @Nour-Mws for spoting this! 
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/939043f6fe59516cda60b0d13227d77f50fb114b.